pagopa/io-app
# fix: [IOBP-482] Adapted actual payment transaction apis with LV

## Short description
This PR integrates LV to the last step into the as-is payment transaction summary;

## List of changes proposed in this pull request
- Added the `withRefreshApiCall` handler to the payment verification API requests into relative sagas

## How to test
- Enable the login veloce feature on the dev-server and decrease the session time.
- Start a payment and let the session to expire
- You should be able to see the refresh session flow as soon as the session expires while starting a payment.
